reference-architecture/azure-ansible/README.md

```diff
@@ -0,0 +1,75 @@
+# RedHat Openshift Enterprise cluster on Azure
+
+When creating the RedHat Openshift Enterprise Cluster on Azure, you will need a SSH RSA key for access.
+
+## SSH Key Generation
+
+1. Windows - https://www.digitalocean.com/community/tutorials/how-to-create-ssh-keys-with-putty-to-connect-to-a-vps
+2. Linux - https://help.ubuntu.com/community/SSH/OpenSSH/Keys#Generating_RSA_Keys
```

Comment:
sad part is the ubuntu one is clearer :( 
and I dont like the digital ocean one much better, but putty not exactly documented in our docs.
Im still agravated that subscription-manager disable * actually is a No-Op, yet its in every install guide. In my case easy to fix, but hate to think the pain it causes.
Man its must be close to 1AM for u.